### DISCUSSION TITLE
Copy Raw Code with Safari is broken

### DISCUSSION THREAD

**Discussion by karlcow** [community/community#65877] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

1. With Safari on macOS
2. Go to https://github.com/WebKit/WebKit/blob/main/Source/WebCore/page/PageInlines.h
3. In the top right corner of the code area ![Screenshot 2023-09-04 at 17 47 19](https://github.com/community/community/assets/505230/b9153f87-f15b-4420-9521-3d8b2fbbfee5)
4. Click on the icon for copying code

Result:
A message is displayed with something went wrong.

On https://github.com/webcompat/web-bugs/issues/126583 Started an attempt to diagnose, but not sure if the devtools are getting into the way of the diagnosis. First I thought the async fetch was the cause, but it doesn't seem. When the fetch is working it is still failing to properly populate the clipboard.

I tried to do a regression range for WebKit but without success, the bug reproduces on older version of WebKit too.

**Comment by NickNaskida** [comment]
I have been having this issue for the last 3 weeks. Please fix this ASAP. 😄

**Comment by verbess** [comment]
Same problem, and it's working fine on Chrome.

**Comment by AdamShwert** [comment, marked as answer]
@karlcow A fix for this should now be deployed. Please let me know if the issue still occurs!

**Reply by verbess** [reply]
Thank you, it's working.

**Reply by karlcow** [reply]
Thanks a lot.

**Reply by aniketnarvekar** [reply]
I'm have the same issue in Safari 18.1.1
